salt/modules/cmdmod.py

```diff
@@ -2965,6 +2964,9 @@ def run_chroot(root,
         arguments. Set to True to use shell features, such as pipes or
         redirection.
 
+    :param list binds: List of directories that will be exported inside
```

Comment:
Should we have a versionadded `Sodium` (i.e. the next major release) here? I don't remember the precise syntax for it.

Reply:
Indeed. Added. I used the version name, instead the number (https://docs.saltstack.com/en/latest/topics/releases/version_numbers.html have a TBD for the version number in Sodium)

Thanks!